### PR TITLE
UHF-248 Liftup image placeholder

### PR DESCRIPTION
**Pull request by khalima**
How to test (if you have no setup etc.):
- `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
- Switch to corresponding branches: 
   - `composer require drupal/hdbt_admin:dev-UHF-248_liftup_image_placeholder && composer require drupal/hdbt:dev-UHF-248_liftup_image_placeholder`
- Run `make new`
- Log in and go to https://hel-platform.docker.sh/admin/tools/site-settings
- Check that you can
   - change the default theme color for the site
   - change the default image placeholder icon for the site
   - change the koro (wave motif) for the site
- Save the changes and go to https://hel-platform.docker.sh/admin/appearance/hdbt/component-library
   - Open the "Card - image placeholder" component and check that the placeholder is the one you've selected from Site settings

Also check these PRs:
- https://github.com/City-of-Helsinki/drupal-hdbt/pull/33
- https://github.com/City-of-Helsinki/drupal-helfi/pull/38

### CODE REVIEW

File: modules/hdbt_admin_tools/config/schema/hdbt_admin_tools.schema.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ hdbt_admin_tools.site_settings:
   type: config_object
   label: 'Site settings'
   mapping:
-    koro_settings:
+    site_settings:
       label: 'Koro settings'
```
Comment (teroelonen) [inline modules/hdbt_admin_tools/config/schema/hdbt_admin_tools.schema.yml:6]: We could change this.

Comment (teroelonen) [reply modules/hdbt_admin_tools/config/schema/hdbt_admin_tools.schema.yml:6]: And the "Koro" on line 10 as well :)

**Review by teroelonen — CHANGES_REQUESTED** [review]
This work is absolutely stunning. Worked like a charm from the start! Just one naming thing that I found that I would like to change :)

File: modules/hdbt_admin_tools/config/schema/hdbt_admin_tools.schema.yml (hunk 1 above)
Comment (khalima) [reply modules/hdbt_admin_tools/config/schema/hdbt_admin_tools.schema.yml]: Good eyes you got there! Fixed.

**Pull request merged**